test/test_corpus.cc

```diff
@@ -507,7 +508,7 @@ TEST_P(ExpectationTest, PerPhaseTest) { // NOLINT
     // resolver
     trees = resolver::Resolver::runTreePasses(ctx, move(newTrees));
 
-    for (auto &resolvedTree : newTrees) {
+    for (auto &resolvedTree : trees) {
```

Comment:
We just moved `newTrees` in the previous statement, so we were accessing a moved vector! clangd told me about this.